Doc/library/configparser.rst

```diff
@@ -1036,6 +1036,13 @@ ConfigParser Objects
       files using :meth:`read_file` before calling :meth:`read` for any
       optional files::
 
+      Raise :exc:`ParsingError` instead of :exc:`AttributeError`
+      when the configuration contains a key without a corresponding value.
+      This change is intended to handle scenarios where a key lacks a value
+      in a configuration which contains extra space or newline within a section.
+      Please note that it's not recommended to have extra spaces
+      or blank lines within sections of the configuration.
+
          import configparser, os
 
          config = configparser.ConfigParser()
```

Comment:
I can see the indentation issue here only, but in my local fork, it is not.

Reply:
`make check` passes (linting), but `make html` (building) fails.

Reply:
yes, got the issue.